packages/api-clients/api-codegen-preset/README.md

```diff
@@ -333,3 +333,38 @@ To make your development flow faster, you can pass in the `--watch` flag to upda
 ```sh
 npm run graphql-codegen -- --watch
 ```
+
+### Using Generated Types with Direct API Access
+
+With Shopify App Bridge, you can use [Direct API Access](https://shopify.dev/docs/api/app-bridge-library#direct-api-access) to make API requests from your apps frontend directly to the Admin API.
+
+Because you are using the Fetch API directly, the responses cannot be automatically typed.
```

Comment:
Reading this seems to suggest that you are on your own.  Is it worth elaborating? E.g:

```
Because you are using the Fetch API directly, the responses cannot be automatically typed.  Here is how to manually type them using the automatically generated types
```